src/mainframe/endpoints/report.py

```diff
@@ -188,31 +189,51 @@ def report_package(
                     f"`{name}@{version}` has no matched rules in the database"
                 ),
             )
-            log.debug("Missing additional_information field", tag="missing_additional_information")
+            log.error(
+                "Missing additional_information field", error_message=error.detail, tag="missing_additional_information"
+            )
             raise error
 
     rules_matched.extend(rule.name for rule in row.rules)
 
     additional_information = body.additional_information
 
-    send_report_email(
-        graph_client,  # type: ignore
-        recipient=body.recipient or mainframe_settings.email_recipient,
-        package_name=name,
-        package_version=version,
-        inspector_url=inspector_url,
-        additional_information=additional_information,
-        rules_matched=rules_matched,
-    )
+    if body.use_email is True:
```

Comment:
I would rather not have this feature in Mainframe, to be honest. It also can't be implemented nicely on the Discord bot - there's no way to to toggle options when submitting a report. I think this should be moved to the reporter service and it can simply fall back to emails if the API fails for whatever reason. Or we can still have an option to use the email instead of the API on the reporter's side